### PR TITLE
Portal fix for PHP 8 and CC style

### LINKED ISSUE #4475: Portal broken on PHP 8
### Describe the bug
Fixed several library classes but Profile edit won't render. Renders to white screen with no indication why.

I guess not many are using PHP 8 with portal!

### PR DESCRIPTION
**Pull request by sjpadgett**
More to come

Fixes #4475

#### Short description of what this resolves:
Fails on PHP 8
Stripe credit card dialog

### CODE REVIEW

File: portal/patient/templates/PatientListView.tpl.php [hunk 1]
```diff
@@ -187,7 +187,7 @@
                     <div class="controls inline-inputs">
                     <?php
                   # Generate drop down list for Marital Status
-                    echo generate_select_list('status', 'marital', $row['marital'], xl('Marital Status'), 'Unassigned', "form-control");
+                    echo generate_select_list('status', 'marital', $row['status'], xl('Marital Status'), 'Unassigned', "form-control");
```
Comment (bradymiller) [inline portal/patient/templates/PatientListView.tpl.php:190]: rain-dance

**Comment by sjpadgett** [comment]
Merging soon. Responsive still needs work for IOS but, this is all i'll do for awhile....
@juggernautsei Note!

**Comment by sjpadgett** [comment]
Some screens for viewing pleasure...
image
With using core themes
image
And
image
And the end!

**Comment by stephenwaite** [comment]
southpark_cheering

**Comment by bradymiller** [comment]
2DV

**Comment by juggernautsei** [comment]
@sjpadgett let me know when this gets pushed to the master repo. I just pulled down master repo and these changes are not in the repo yet.

**Comment by sjpadgett** [comment]
@juggernautsei this is in master. note it's merged. also in v6 for patch 3. here is the v6 changes the way they'll be in patch 3. i've tested on both master and v6 with ipad.
[portal-patch3.zip](https://github.com/openemr/openemr/files/6718382/portal-patch3.zip)

**Comment by juggernautsei** [comment]
Ok, I will load these up and let you know.

**Comment by sjpadgett** [comment]
So @juggernautsei How did this turn out?

**Comment by juggernautsei** [comment]
@sjpadgett it turned out fine. I have had no more issues raised about patients not being able to pay. This is for a virtual behavioral health clinic.

**Comment by juggernautsei** [comment]
@sjpadgett 
Where is the database can I go to update the forms we need to change to a required status.
image

**Comment by sjpadgett** [comment]
@juggernautsei check out this thread
https://community.open-emr.org/t/patient-documents-for-portal-and-staff-rework/16303/9?u=sjpadgett

**Pull request merged**